booking/app/views.py

```
@@ -0,0 +1,49 @@
+
+from flask import abort, request
+from flask_restful import Resource, marshal_with
+
+from booking.manager import ItemManager, DataManager
+from . import marshallers
+
+class EventsListAPI(Resource):
+
+    def __init__(self):
+        self.manager = ItemManager()
+
+
+    @marshal_with(marshallers.event_fields)
+    def get(self):
+        return self.manager.list()
+
+
+    @marshal_with(marshallers.event_fields)
+    def post(self):
+        return self.manager.create(**request.json)
+
+class EventDetailAPI(Resource):
+
+    def __init__(self):
+        self.manager = ItemManager()
+
+    @marshal_with(marshallers.event_fields)
+    def get(self, event_id):
+        try:
+            return ItemManager().get(event_id)
+        except:
```

Bare except clause shouldn't be used, except StorageError specifically instead.